test/unittest/unittest_util_func.cpp

```diff
@@ -136,7 +172,7 @@ TEST(nntrainer_util_func, sigmoid_01_p) {
   ASSERT_NE(nullptr, indata);
 
   for (int i = 0; i < batch * height * width; ++i) {
-    if ((data[i] - (float)(1 / (1 + exp(-indata[i])))) > tolerance) {
+    if (fabs(data[i] - answer[i]) > tolerance) {
```

Comment:
how about using `EXPECT_NEAR(val1, val2, abs_error);`? 
I have never used it, but I think the code will be simpler.
reference link ; https://github.com/google/googletest/blob/master/googletest/docs/advanced.md#floating-point-macros

Reply:
Thank you for the suggestion! I changed it :)